veps/sig-network/passt/passt-beta.md

```diff
@@ -0,0 +1,174 @@
+thus taking full ownership over its code, simplifying the architecture and implementation a great deal, and treating it as a trustable "first class citizen" in the code.
+This change, targeted at KubeVirt release 1.7, will also promote the feature stage from Alpha to Beta, and will be conditioned by a new Feature Gate `PasstBindingEnabled`.
+
+> [!NOTE] 
+> `passt` used to be a core network binding, and was [removed](https://github.com/kubevirt/kubevirt/pull/11915) in KubeVirt release 1.3, as Network Binding Plugins 
+functionality was introduced.   
+
+## Motivation
+
+VEP #20 describes the benefits of passt in detail, it commits to migrate the functionality to the core for the Beta release, and KubeVirt will benefit
+from accelerating its graduation as early as possible.
+Technically, the functionality that was implemented for VEP#20 Alpha is very complex, the architecture is far from straight-forward, and having the implementation
+distributed to many different areas of the product is worsened as a consequence of the plugin implementation.
+Specifically, product areas that can be consolidated into concise core code include:
+- Sidecar container along with associated GRPC communication + its build/publish process
+- CNI plugin + its build/publish process
+- DaemonSet that deploys the CNI plugin
+- A Network Attachment Definition CR
+- KubeVirt Configuration 
+
+E2e tests depend on all of the above.
+In addition, since the current implementation is a plugin, the core code cannot refer to it specifically, and instead [looks
+for hints](https://github.com/kubevirt/kubevirt/blob/release-1.6/pkg/network/passt/repair.go#L122) of `passt` as a workaround for not being able to reference the API.
+
+
+## Goals
+
+- Simplification and reduction of code and architecture complexity
+- Simplified, documented API for users
+- Simplification of operability and testing
+
+## Non Goals
+
+- Functional changes.
+- Having an upgrade path from the passt binding plugin to the proposed core binding without VM restart.
+
+## Definition of Users
+
+- VM Owner
+- Cluster Admin
+
+## User Stories
+- As a Cluster Admin I would like to deal with fewer moving parts and dependencies for my network bindings to function.   
+- As a VM Owner I would prefer to have a straightforward, documented VM API that natively supports the bindings that I frequently use.   
+
+## Repos
+
+- kubevirt/kubevirt
+
+## Design
+
+### API Changes 
+The passt core network binding will be enabled by a new InterfaceBindingMethod member:
+```go
+type InterfaceBindingMethod struct {
+	...
+	PasstBinding InterfacePasstBinding `json:"passtBinding,omitempty"`
+}
+
+type InterfacePasstBinding struct{}
+```
+> [!NOTE]
+> DeprecatedPasstBinding is still a member of the struct. It represents a previous generation of passt which has been deprecated in release 1.3
+
+A new Feature Gate `PasstBindingEnabled` will be introduced, to condition the use of this binding.
```

Comment:
What name will appear in the yaml/json format?
The variable or enum name is not that important.

BTW, why did you add the `Enabled` suffix?